target/linux/ramips/dts/mt7621_maginon_mc-1200ac.dts

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "maginon,mc-1200ac", "mediatek,mt7621-soc";
+	model = "Maginon MC-1200AC";
+    
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+	
+	aliases {
+		led-running = &led_blue_top;
+		led-boot = &led_red_top;
+		led-failsafe = &led_red_top;
+		led-upgrade = &led_red_top;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_blue_top: led_blue {
+			function = LED_FUNCTION_STATUS;
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_red_top: led_red {
+			function = LED_FUNCTION_STATUS;
+			color = <LED_COLOR_ID_RED>;
+			gpios = <&gpio 17 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset_button: reset {
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+		touch_button: touch {
```

Comment:
Please insert new empty line between two nodes. And please add button name labels.
```suggestion
		touch {
		label = "touch";
```